### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by ishapatell1**
Screenshot (96)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -15,7 +15,41 @@ html {
   font-size: 62.5%;
   scroll-behavior: smooth;
 }
+.logo {
+  font-size: 3.5rem;
+  font-style: italic;
+  font-weight: 800;
+  color: #fff;
```
Comment (Karan-Bharti) [inline style.css:22]: Try to use css variable properties for color

File: style.css [hunk 2]
```diff
@@ -15,7 +15,41 @@ html {
...
+  position: relative;
```
Comment (Karan-Bharti) [inline style.css:23]: Use css variable for color code

**Review by Karan-Bharti — APPROVED** [review]
I want all lines where You had used colors...there requires use of css variable as it is a good code practice

**Review by spraveenofficial — APPROVED** [review]
Make some gap between input boxes. Else looks good.

**Pull request merged**